Makefile.in

```diff
@@ -195,11 +195,16 @@ ifeq ($(shell uname), Darwin)
 endif
 
 #
-# All translation message catalogs
+# All translation message catalogs, filter files based on LINGUAS.
 #
 TRANSLATIONS_SRC := $(wildcard po/*.po)
-ifdef HAVE_GETTEXT
+ifeq ($(HAVE_GETTEXT), 1)
 	TRANSLATIONS := $(TRANSLATIONS_SRC:.po=.gmo)
+ifdef LINGUAS
```

Comment:
it just compares 2 whitespace delimited file lists and leaves only what LINGUAS wants. even if the lists ends up empty there is no regression, as it's technically the same as building with empty _TRANSLATIONS_ variable.
since Makefile stores relative filepaths in _TRANSLATIONS_ variable I had to do some string substitution to replace _LINGUAS="en de"_ with _TRANSLATIONS_WANTED="po/en.gmo po/de.gmo"_